src/createNumbersGenerator.js

```diff
@@ -26,7 +26,20 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let counter = 0;
+
+  return () => {
+    if (numbers === undefined) {
+      return undefined;
+    }
+    for (let i = counter; i < numbers.length; i++) {
+      counter++;
+      if (min === undefined || max === undefined
```

Comment:
good, but if we will receive only `max` or `min`?

you can solve this problem with another check or add default values to min and max